experiments/is_my_project_on_track.md

```diff
@@ -0,0 +1,15 @@
+- are you hacking?
+    + do you have a decision to make?
+        * Stop and decide
+    + do you have a PHD?
+        * What's a PHD =>  
+        go do that.
+- Firefox Peer
+- PI-request
+    + requires a peer review
+    + qa plan
+
+
+Realistic date planning.
+
+- QA will discover problems.  They will be on inconvenient 
```

Comment:
Not sure if the "They" refers to "QA" or "problems" here, but either way it is valid and I love it.